srtcore/channel.h

```diff
@@ -60,6 +60,7 @@ modified by
 
 class CChannel
 {
+    void createSocket(int family);
 public:
 
    // XXX There's currently no way to access the socket ID set for
```

Comment:
`void createSocket(int family);` should have 3 space alignment, according to the following section.